.github/workflows/CI.yml

```
@@ -19,11 +19,11 @@ jobs:
           - nightly
           - "1.53.0"
         conf:
-          - { name: "gtk4", features: "v4_2", test_sys: false }
-          - { name: "gsk4", features: "v4_2,broadway,vulkan", test_sys: true }
-          - { name: "gdk4", features: "v4_2", test_sys: true }
-          - { name: "gdk4-wayland", features: "", test_sys: true }
-          - { name: "gdk4-x11", features: "", test_sys: false }
+          - { name: "gtk4", features: "v4_2,v4_4", test_sys: false }
```

```suggestion
          - { name: "gtk4", features: "v4_4", test_sys: false }
```
You can drop v4_2 from the ones with v4_4

---

ah ok, done